docs/sources/installation/rhel.md

```diff
@@ -47,6 +47,14 @@ running on kernels shipped by the distribution. There are things like namespace
 changes which will cause issues if one decides to step outside that box and run
 non-distro kernel packages.
 
+> **Warning**:
+> Please make sure that your system is up to date by installing updates
+> using `yum update` and rebooting your system.
+> Keeping the system up to date is recommended to ensure your system
+> receives fixes for critical security vulnerabilities and severe bugs.
+> The fixes for severe bugs include fixes for potential kernel panics
```

Comment:
Multiple issues. Suggested revision:

**Important**
Please keep your system up-to-date with `yum update`, rebooting as needed. Keeping your system updated ensures that critical security vulnerabilities and severe bugs (such as those found in kernel 2.6.32) are fixed.